doc/dev/release.rst

```diff
@@ -67,12 +67,9 @@ release makers, following this checklist by MRC.
 #. (Optional) Check for updates to versioneer::
 
         pip install --upgrade versioneer
-        versioneer-installer
+        versioneer install
+        git diff --staged
 
-        git diff
-
-        ./setup.py versioneer
```

Comment:
Whatever it was, it is not needed anymore (I tried to dig in git history, it is not used for at least 2 years). The updated version of the [screed release checklist][1] is way shorter on the versioneer installation step.

[1]: https://screed.readthedocs.io/en/v1.0/dev/release-checklist.html#getting-started